### PR TITLE
SA-202, return full http response error message

### PR DESCRIPTION
**Pull request by vidhutsingh**
Signed-off-by: vidhut-kumar-singh <vidhut-kumar.singh@hpe.com>

### CODE REVIEW

File: pkg/client/group_mgmt_client.go [hunk 1]
```diff
@@ -393,7 +393,7 @@ func processAsyncResponse(client *GroupMgmtClient, body []byte) (interface{}, er
 			}
 		}
 		if len(jobId) == 0 {
-			return nil, fmt.Errorf("http response error: status (202), failed to get the job id")
+			return nil, fmt.Errorf("http response error: status (202), failed to get the job id, message: %v",errResp)
```
Comment (rkumpf) [inline pkg/client/group_mgmt_client.go]: I suggest changing "failed to get the job id" to "but no job ID returned".  Also change "message" to "messages" for consistency with the other errors.

File: pkg/client/group_mgmt_client.go [hunk 2]
```diff
@@ -393,7 +393,7 @@ func processAsyncResponse(client *GroupMgmtClient, body []byte) (interface{}, er
 			}
 		}
 		if len(jobId) == 0 {
```
Comment (rkumpf) [inline pkg/client/group_mgmt_client.go:395]: At line 375, errResp is constructed with no separator between the messages.  I'm curious to see how it looks with multiple messages.

Comment (vidhutsingh) [reply pkg/client/group_mgmt_client.go:395]: if you notice there is a default separator in the message itself and we don't want to add a custom separator.

File: pkg/client/group_mgmt_client.go (hunk 1 above)
Comment (vidhutsingh) [reply pkg/client/group_mgmt_client.go]: sure

File: pkg/client/group_mgmt_client.go (hunk 2 above)
Comment (rkumpf) [reply pkg/client/group_mgmt_client.go:395]: I can't tell that there is a separator in the message, but I'll take your word for it.

**Review by rkumpf — APPROVED** [review]

**Comment by vidhutsingh** [comment]
thanks rkumpf

**Pull request merged**